documentation/RLC-quickstart.md

```diff
@@ -99,19 +108,27 @@ We are working on to automatically generate everything right now, but currently
     rushx pack
     ```
 
-    But we still need to add some tests for it.
+    The generated code is not enough to release at once and you need to update it for better usage experience.
+
+# Improve README.md document
+
+A minimal README.md is generated by typescript emitter and you could improve it with more meaningful information to let customers know more about your service.
```

Comment:
```suggestion
A minimal README.md is generated by TypeScript emitter and you could improve it with more meaningful information to let customers know more about your service.
```
maybe check the whole PR for word like typescript or javascript and change them into TypeScript or JavaScript